### ISSUE TITLE
Add to Steam w/o Install

### ISSUE THREAD

**Issue by reverendj1** [Heroic-Games-Launcher/HeroicGamesLauncher#3378]
### Problem description

I have a Steam Deck and primarily use Steam. I mostly use Heroic to install games from other stores. It would be nice to be able to add games to Steam, without first installing them.

My personal use case is that I keep a backlog of games in Steam as a category, so I just pick a random game from my backlog (using Deck roulette) when I finish a game and install it. It'd be nice to include my Heroic games here. Right now, my workaround is to install the game which creates the Steam shortcut, then uninstall it. This is not ideal, especially with larger games.

### Feature description

Basically just remove the restriction for a game being installed before the "Add to Steam" shows up in the options. If a game is not installed already, Heroic just pops up with the install options, so it already handles the situation gracefully.

**Comment by TabulateJarl8** [comment]
I may be wrong, but I believe the way that it works requires games to be installed. Since Heroic is installing Epic and GOG games, how will Steam know where to download the games once you choose to install them? Steam doesn't have support for downloading from other stores, Heroic just adds the physical game executable to Steam for Steam to launch if that makes sense. Others feel free to correct me if I'm wrong, but I hope this helped

**Comment by reverendj1** [comment]
As I mentioned, the games aren't currently required to be installed. Heroic launches the games, even when using a shortcut. If you install a game in Heroic, it will create the shortcut in Steam. If you then uninstall the game in Heroic, it leaves the Steam shortcut. Launching that, Heroic will just prompt to install the game.

**Comment by flavioislima** [comment]
>  If you then uninstall the game in Heroic, it leaves the Steam shortcut.

This should be true only if you have steam opened or uninstalled the game on game mode on the steam deck, because then Heroic cannot edit the Steam shortcuts file so it cannot either add or remove shortcuts.

**Comment by Undeemiss** [comment]
As a user with a pretty large collection of games obtained for free through Epic Games, GOG, and Prime Gaming, but who struggles to actually remember to search my libraries other than Steam, I would really appreciate it if there was an option to add dummy listings for games that aren't installed, but are available through Heroic. It doesn't even need to actually work, I just want to see the title and some basic metadata, so that I'm reminded the games are available to me. I can understand if there's hesitancy to support my use case, but I just wanted to voice my thoughts on the matter.
Particularly worth noting is that I think it would be really nice if there were an option to do this by default when a new game is detected in my library, or at least an option to do it in bulk; I have over 200 games I can only launch through Heroic at this point.

**Comment by NonKurow** [comment]
Add (not installed) games to Steam and/or Don't delete Steam shortcut after uninstalling from Heroic

(already suggested this in discord server)

**Comment by PrivacyFriendlyMuffins** [comment]
I too would like this option!

Before switching to Linux from Windows, I used Playnite to organize all my games. I have some games on disk from the old days and with Playnite I would add an entry and when I wanted to play a game that wasn't installed but not on GOG or Steam or other storefronts I would simply install it, then change the install directory to the one that the disk created.

This would be a great option to have the same ability: the ability to use Heroic as a game *library* regardless of format!
